lib/modules/manager/poetry/__snapshots__/extract.spec.ts.snap

```diff
@@ -504,7 +493,7 @@ exports[`modules/manager/poetry/extract extractPackageFile() extracts multiple d
       "nestedVersion": false,
     },
     "packageName": "dev-dep2",
-    "skipReason": "unspecified-version",
+    "skipReason": "invalid-version",
```

Comment:
Isn't unspecified more accurate than invalid?

Reply:
In this particular case `currentValue` equals to `Invalid version.`, so I think `invalid-version` is more accurate.

Reply:
Isn't it missing, rather than in valid?

Reply:
<img width="408" alt="image" src="https://github.com/renovatebot/renovate/assets/1239644/2f60540a-ad96-429c-8854-1406c3722045">